src/api/app/views/status_messages/index.xml.builder

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Is there a reason for the extra line here?

Reply:
No, it's just there because we copied the file over. I'll remove it.